src/main/webapp/WEB-INF/views/index.jsp

```diff
@@ -1,9 +1,10 @@
 <%@ page contentType="text/html;charset=UTF-8" language="java" %>
 <html>
     <head>
-        <title>My team</title>
+        <title>Taxi service</title>
     </head>
     <body>
-        <h1>Hello, mates!!!!</h1>
+        <h1>Taxi Service</h1>
+        <jsp:include page="/WEB-INF/views/head_menu.jsp"></jsp:include>
```

Comment:
in this case you display an unnecessary link to main page on main page

Reply:
I don't think it is a problem. Ive seen the same a lot of times on other sites